SocketHttpListener/HttpResponse.cs

```diff
@@ -115,10 +115,6 @@ public override string ToString()
 
             output.Append(CrLf);
 
-            var entity = EntityBody;
```

Comment:
Why are you removing this?

Reply:
Unused.

Reply:
Because it's never set, will always be of length 0